### ISSUE TITLE
Can not handle varbinary (ipv4 bin data)

### ISSUE THREAD

**Issue by zloidemon** [tarantool/libslave#3]
I got error like this binary ipv4 data

```
2017-04-07 13:02:20] E Error in reading binlogs: Slave::create_table(): cannot find collate '' from field originating_ip type varbinary(4) in collate info map
```

**Comment by deadbabe** [comment]
Should be fixed with #5

**Comment by zloidemon** [comment]
reopen https://github.com/tarantool/libslave/issues/7
